svelte/package.json

```diff
@@ -0,0 +1,69 @@
+{
+	"name": "uplot-svelte",
+	"version": "1.1.5",
+	"description": "Svelte wrapper for uPlot that allows you to work with charts declaratively inside your favorite framework",
+	"author": "Kiril Panayotov <contact@zakrok.dev> (https://zakrok.dev)",
+	"contributors": [
+		{
+			"name": "Sergey Kalinichev",
+			"email": "kalinichev.so.0@gmail.com"
+		}
+	],
+	"license": "MIT",
+	"homepage": "https://github.com/skalinichev/uplot-wrappers#readme",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/skalinichev/uplot-wrappers.git"
+	},
+	"bugs": {
+		"url": "https://github.com/skalinichev/uplot-wrappers/issues"
+	},
+	"keywords": [
+		"Charts",
+		"uPlot",
+		"Svelte",
+		"SvelteKit",
+		"Wrapper"
+	],
+	"scripts": {
+		"dev": "vite dev",
+		"build": "vite build && npm run package",
+		"preview": "vite preview",
+		"package": "svelte-kit sync && svelte-package && publint",
+		"prepublishOnly": "npm run package",
+		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch"
+	},
+	"exports": {
+		".": {
+			"types": "./dist/index.d.ts",
+			"svelte": "./dist/index.js"
+		}
+	},
+	"files": [
+		"dist",
+		"!dist/**/*.test.*",
+		"!dist/**/*.spec.*"
+	],
+	"dependencies": {},
+	"peerDependencies": {
+		"svelte": "^4.0.0",
+		"uplot": "^1.6.30"
+	},
+	"devDependencies": {
+		"@sveltejs/adapter-auto": "^3.0.0",
+		"@sveltejs/kit": "^2.0.0",
+		"@sveltejs/package": "^2.0.0",
+		"@sveltejs/vite-plugin-svelte": "^3.0.0",
+		"publint": "^0.1.9",
```

Comment:
I'm not expecting there to be many changes in the package.json file. Can we drop it? The idea is to reduce the dependencies to a bare minimum. If we decide to add it in the future it should be placed in the root directory anyway.